wram.asm

```diff
@@ -1565,7 +1565,7 @@ wDaysSince:: db
 
 SECTION "WRAM 1", WRAMX
 
-wd000:: ds 1
+wDecompressBuffer:: ds 1
```

Comment:
should this be a union?

Reply:
No, there are no uses of unidentified `wd000` left.

Reply:
pretty sure wDecompressBuffer is not 1 byte large

Reply:
Okay, I see. You could, but there's a big union right after it at d002, and every NEXTU would need a `ds 2` at the beginning. How about a comment noting that `wDecompressBuffer` is larger than that? (Max size: `GBCOnlyGFX` takes $540 bytes uncompressed, and ~~I think `LZ_170d16` takes $c10 in mobile_5c.asm~~. Oh, mobile_5c.asm should have used `wDecompressScratch`, not `wDecompressBuffer`. So yeah, $540 bytes, and `wGBCOnlyDecompressBuffer` would be a more accurate name.)